omop_file_validator.py

```diff
@@ -388,27 +390,22 @@ def run_checks(file_path, f):
 
                             for idx, value in df[submission_column].iteritems(
                             ):
+                                # convert pd nan values to empty string
+                                df = df.replace(np.nan, '')
                                 if not any(
                                         list(
                                             map(
                                                 lambda pattern:
                                                 date_format_valid(
-                                                    pattern, str(value), fmt),
-                                                patterns))):
-                                    invalid_indices.append(idx + 1)
-                                    invalid_date_strings.append(str(value))
-
-                            invalid_indices = [
-                                str(idx) for idx in invalid_indices
-                            ]
-                            if invalid_indices:
-                                line_num_str = 'line numbers' if len(
-                                    invalid_indices) > 1 else 'line number'
-                                e = dict(
-                                    message=
-                                    f"{err_msg}: {line_num_str} ({','.join(invalid_indices)})",
-                                    column_name=submission_column)
-                                result['errors'].append(e)
+                                                    pattern, str(value), fmt), patterns))):
+                                    if value == '' and not meta_column_required:
+                                        continue
+                                    else:
```

Comment:
This can probably be simplified too
```suggestion
                                    if not (value == '' and not meta_column_required):
```

Reply:
Adjusted this as well, thanks for that!